decentraland/unity-renderer
# fix: dynamic size of channel member count container

## What does this PR change?

The member count indicator inside a channel window has now a dynamic size depending on the amount of members.
Also minor visual refinement over the "cant create new channel because you reached the limit" popup.

## How to test the changes?

1. Go to the channel browser and join many channels
2. Open the different channels from the conversation list
3. In all the cases, the amount of members at the header of the window should be displayed correctly

## Our Code Review Standards

https://github.com/decentraland/unity-renderer/blob/master/docs/code-review-standards.md
